scripts/backup-restore.sh

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/env bash
+
+set -e
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+usage() {
+  echo "Usage: $0 <action> <target>"
+  echo "Actions: backup | restore"
+  echo "Targets: opensearch | harbor | rclone | velero"
+  exit 1
+}
+
+# Sets a timestamp log before every message, also colors it purple to make it easier to distinct from rest of the text because why not
+PURPLE='\033[0;35m'
+NOCOLOR='\033[0m'
+log() {
+  echo -e "[${PURPLE}$(date --utc +'%Y-%m-%d %H:%M:%S')${NOCOLOR}]: $1"
```

Comment:
Suggestion: Do we want to keep this local time? Since this will only be shown to the user running the script?

```suggestion
  echo -e "[${PURPLE}$(date +'%Y-%m-%d %H:%M:%S')${NOCOLOR}]: $1"
```